types/ntlm-client/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for ntlm-client 0.1
+// Project: https://github.com/clncln1/node-ntlm-client
+// Definitions by: Manuel Borrajo <https://github.com/borrajo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import "node";
```

Comment:
This import is not needed.

Reply:
if i remove the import, running the lint throws me this error:

`ERROR: 17:17  expect  TypeScript@next compile error: 
Cannot find name 'Buffer'. Do you need to install type definitions for node? Try npm i @types/node and then add node to the types field in your tsconfig.`

Then add 'node' in types in tsconfig 
`"types": [ "node" ]`
And when I run the lint again, error:
`Error: Expected compilerOptions["types"] === []`

So the only way I found to solve it was by `import "node"`. 
Can you think of any other way to solve this?

Reply:
If thats the case you need to change this to `/// <reference types="node"/>`